src/libYARP_OS/src/Node.cpp

```diff
@@ -549,13 +549,22 @@ Node::Node() :
 Node::Node(const ConstString& name) :
         mPriv(new Helper)
 {
+    ConstString _name;
+    if(name.at(0) != '/')
+    {
+        _name = "/"+name;
+    }
+    else
+    {
+        _name = name;
+    }
```

Comment:
Please, make a separate PR for this change, even if it is very small it is a change of behaviour in YARP_OS, and I don't want this to be hidden in a "RGBDbugfix" pull request...